bin/dzil

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/perl
-package
-  dzil;
```

Comment:
This is intentional and affects PAUSE's indexing of the package, so should not be changed.

Reply:
Could you please elaborate more on the PAUSE indexing? Thanks.

Reply:
From the PAUSE operating model (https://pause.perl.org/pause/query?ACTION=pause_operating_model)

> In the source of the module, if there is a newline character between "package" and the package name, then PAUSE will take this as a hint to not consider the module for indexing (this was a hack put in place before the no_index directive was available, but given it is used by many distributions on CPAN, it will be respected for the foreseeable future).

Reply:
Thanks, I see now, perhaps it's time to switch to the `no_index` directive? :-)